web/src/main/kotlin/no/nav/su/se/bakover/web/routes/sak/SakRoutes.kt

```diff
@@ -29,75 +29,77 @@ internal const val sakPath = "/saker"
+                                    .map {
+                                        call.audit(fnr, AuditLogEvent.Action.ACCESS, null)
+                                        call.svar(Resultat.json(OK, serialize(it.toJson())))
+                                    }
+                            },
+                        )
+                    }
+                    body.saksnummer != null -> {
+                        Saksnummer.tryParse(body.saksnummer).fold(
+                            ifLeft = {
+                                call.svar(
+                                    BadRequest.errorJson(
+                                        "${body.saksnummer} er ikke et gyldig saksnummer",
+                                        "saksnummer_ikke_gyldig"
                                     )
-                                }
-                                .map {
-                                    call.audit(fnr, AuditLogEvent.Action.ACCESS, null)
-                                    call.svar(Resultat.json(OK, serialize(it.toJson())))
-                                }
-                        },
-                    )
-                }
-                body.saksnummer != null -> {
-                    Saksnummer.tryParse(body.saksnummer).fold(
-                        ifLeft = {
-                            call.svar(
-                                BadRequest.errorJson(
-                                    "${body.saksnummer} er ikke et gyldig saksnummer",
-                                    "saksnummer_ikke_gyldig"
                                 )
-                            )
-                        },
-                        ifRight = { saksnummer ->
-                            call.svar(
-                                sakService.hentSak(saksnummer).fold(
-                                    {
-                                        NotFound.errorJson(
-                                            "Fant ikke sak med saksnummer: ${body.saksnummer}",
-                                            "fant_ikke_sak"
-                                        )
-                                    },
-                                    {
-                                        call.audit(it.fnr, AuditLogEvent.Action.ACCESS, null)
-                                        Resultat.json(OK, serialize((it.toJson())))
-                                    },
-                                ),
-                            )
-                        },
+                            },
+                            ifRight = { saksnummer ->
+                                call.svar(
+                                    sakService.hentSak(saksnummer).fold(
+                                        {
+                                            NotFound.errorJson(
+                                                "Fant ikke sak med saksnummer: ${body.saksnummer}",
+                                                "fant_ikke_sak"
```

Comment:
Denne vil uansett bli tolket som en fant_ikke_sak i frontend i motsetning til fant_ikke_sak_for_person som mappes til en custom feil i frontend 🤔

Reply:
Skjønner ikke helt hva du mener her 🤔 Frontend har håndtering av `fant_ikke_sak`, men jeg finner ikke noe om `fant_ikke_sak_for_person`.

Reply:
Det var en `code` jeg så noen linjer over (men jeg innser vel at det var linjer du egentlig ikke hadde til hensikt å endre på):

> NotFound.errorJson(
                                                "Fant ikke noen sak for person: ${body.fnr}",
                                                "fant_ikke_sak_for_person",
                                            ),